### PR TITLE
[CPP Onboarding] Implement Stripe account under review

### PR DESCRIPTION
**Pull request by ctarda**
Part of #4611 

This adds an specific error when the Stripe account is still under review.

The error screen:

When tapping Contact us, we should redirect the user to contact support:

Learn more should open a web view:

## How to test
In order to test this in real conditions, it is necessary to have a store associated to an Stripe account that is under review, or...
make `CardPresentPaymentsOnboardingUseCase.checkOnboardingState` return `stripeAccountUnderReview`

1. Navigate to Settings > In-Person Payments
2. Notice the error screen.

Update release notes:
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsStripeAcountReviewView.swift [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+//  InPersonPaymentsStripeAcountReview.swift
+//  WooCommerce
+//
+//  Created by Cesar Tardaguila on 2021-08-10.
+//  Copyright © 2021 Automattic. All rights reserved.
+//
+
+import SwiftUI
+
+struct InPersonPaymentsStripeAcountReviewView: View {
```
Comment (koke) [inline WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsStripeAcountReviewView.swift]: This is a small nitpick that I missed in previous reviews, but it'd be good for consistency.

I'm not using the `View` suffix for SwiftUI views (even though I'm keeping it for the file name). None of the system views have it (`Text`, `Image`,...) so I do the same for the custom ones.

File: WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsStripeAcountReviewView.swift [hunk 2]
```diff
@@ -0,0 +1,53 @@
+//
+//  InPersonPaymentsStripeAcountReview.swift
+//  WooCommerce
+//
+//  Created by Cesar Tardaguila on 2021-08-10.
+//  Copyright © 2021 Automattic. All rights reserved.
+//
+
```
Comment (koke) [inline WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsStripeAcountReviewView.swift]:
```suggestion
```

IIRC, we don't do file headers. I wish I could make Xcode learn that 😅

**Review by koke — APPROVED** [review]
I left a couple of suggestions but it looks good. I have to go now so I can't properly test, but I'll trust your testing on this one 🙈

File: WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsStripeAcountReviewView.swift (hunk 2 above)
Comment (ctarda) [reply WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsStripeAcountReviewView.swift]: Argh, I hate when I forget to remove these. Sorry!

File: WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsStripeAcountReviewView.swift (hunk 1 above)
Comment (ctarda) [reply WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsStripeAcountReviewView.swift]: Renamed. I will push an update to rename also some of the other pre-existing views.

**Comment by ctarda** [comment]
Thanks @koke . I have renamed some of the other views we merged yesterday, I will merge after CI, and then merge develop back into #4761

**Pull request merged**